### PR TITLE
fix: meeting url missing in workflow email

### LINKED ISSUE #15789: [CAL-3970] Workflow - event rescheduled doesn't include Meeting URL varialbe
[https://cal.com/team/pricer/badanie-rynku-lokalnego](https://cal.com/team/pricer/badanie-rynku-lokalnego)

Happens only for team events, most likely because it changes the host of the meeting

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-3970](https://linear.app/calcom/issue/CAL-3970/workflow-event-rescheduled-doesnt-include-meeting-url-varialbe)</sub>

### PR DESCRIPTION
**Pull request by CarinaWolli**
## What does this PR do?

Fixes that sometimes meeting URL from the {MEETING_URL} variable was missing in workflow emails. Metadata with the videoCallUrl was missing in several places 

- Fixes #15789
- Fixes CAL-3970

### CODE REVIEW

File: packages/trpc/server/routers/viewer/workflows/util.ts [hunk 1]
```diff
@@ -751,6 +752,7 @@ export async function scheduleBookingReminders(
           schedulingType: booking.eventType?.schedulingType,
           hosts: booking.eventType?.hosts,
         },
+        metadata: booking.metadata,
```
Comment (joeauyeung) [inline packages/trpc/server/routers/viewer/workflows/util.ts:755]: Why is metadata being added here?

**Review by joeauyeung — CHANGES_REQUESTED** [review]
While this does fix the issue to me it feels like a bandaid solution. I wonder if we can get the video call URL inside of the workflow calls from the event that's passed. That way in the future someone doesn't forget to include it.

**Comment by CarinaWolli** [comment]
> While this does fix the issue to me it feels like a bandaid solution. I wonder if we can get the video call URL inside of the workflow calls from the event that's passed. That way in the future someone doesn't forget to include it.

Agree, can we still merge this for now and revisit it later?

**Review by joeauyeung — APPROVED** [review]
Talked with Carina about this. Let's merge for now as customers are waiting.

**Comment by CarinaWolli** [comment]
> Talked with Carina about this. Let's merge for now as customers are waiting.

I have a follow up issue for this assigned to me: https://github.com/calcom/cal.com/issues/16491

**Pull request merged**